recipes/avahi-mdnsresponder/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "system":
+    folder: system
```

Comment:
As @Croydon has pointing out, is it the good approach? I'm not confortable with adding a new system library while source code is available.

Reply:
I started work on an avahi recipe from the source package. The configure script has 25 `--enable-xxx` options and a further 10 arguments for those options. It will need a new recipe for **libdaemon** for sure, and several other new recipes if all the options are ultimately to be supported.

Reply:
Consumers of avahi are already using the system package manager, which goes the extra mile to setup the startup scripts for the daemon. Which could be added as a deploy step in downstream recipes to be installed but I am not sure how that would work in CCI.

Even if we build from source we won't be able to correctly install the daemon for that platform.
This give Consumers the option to build and deploy against a target platform and not have to re-invite too many things.

Having the recipe would be a huge advantage to "us" because it blends so nicely with the solutions we've already deployed and are working for end-users.

***

We all agree from source is ideal but in some cases, id argue like this, offer a few key advantages.
This does not prevent us from adding it in the future.